### PR TITLE
Sync GH with core

### PR DESCRIPTION
**Pull request by MaggieCabrera**
**Description**

This PR will add the changes made during the review process of https://github.com/WordPress/wordpress-develop/pull/5283

Let's not merge this until Beta 1 is released later today in case we need to add last minute fixes

### CODE REVIEW

**Comment by MaggieCabrera** [comment]
This needs some more updating with the last minute changes made yesterday (spacing)

**Comment by MaggieCabrera** [comment]
This now has all the changes made in core. Since this code is already live, it just needs a ✅ and merging.

File: patterns/footer-portfolio.php [hunk 1]
```diff
@@ -76,7 +76,7 @@
 			<!-- wp:group {"style":{"spacing":{"blockGap":"6px"}},"layout":{"type":"flex","flexWrap":"wrap"}} -->
 			<div class="wp-block-group">
 				<!-- wp:paragraph {"fontSize":"small"} -->
-				<p class="has-small-font-size"><?php esc_html_e( '© 2024', 'twentytwentyfour' ); ?></p>
+				<p class="has-small-font-size"><?php esc_html_e( '&copy; 2024', 'twentytwentyfour' ); ?></p>
```
Comment (luminuu) [inline patterns/footer-portfolio.php:79]: Shall we bump this to 2023?

File: readme.txt [hunk 2]
```diff
@@ -1,6 +1,6 @@
 === Twenty Twenty-Four ===
 Contributors: wordpressdotorg
-Requires at least: 6.4
+Requires at least: 6.2
```
Comment (luminuu) [inline readme.txt:3]: Why is this down to 6.2? All previous default themes had the requires at least to the version it was shipped with.

File: style.css [hunk 3]
```diff
@@ -4,7 +4,7 @@ Theme URI: https://wordpress.org/themes/twentytwentyfour
 Author: the WordPress team
 Author URI: https://wordpress.org
 Description: Twenty Twenty-Four is designed to be flexible, versatile and applicable to any website. Its collection of templates and patterns tailor to different needs, such as presenting a business, blogging and writing or showcasing work. A multitude of possibilities open up with just a few adjustments to color and typography. Twenty Twenty-Four comes with style variations and full page designs to help speed up the site building process, is fully compatible with the site editor, and takes advantage of new design tools introduced in WordPress 6.4.
-Requires at least: 6.0
+Requires at least: 6.2
```
Comment (luminuu) [inline style.css:7]: See above

**Review by luminuu — CHANGES_REQUESTED** [review]
Just a few minor things to check on

File: patterns/footer-portfolio.php (hunk 1 above)
Comment (MaggieCabrera) [reply patterns/footer-portfolio.php:79]: There is an open [issue](https://github.com/WordPress/twentytwentyfour/issues/510) for this.Let's open a different PR for that, this one is just to bring the repo inline with what's on core right now, we can iterate on that and it's clearer when we need to move our changes to core for beta 2 if they are made outside of this PR

File: readme.txt (hunk 2 above)
Comment (MaggieCabrera) [reply readme.txt:3]: I agree, but this is just what's on core right now. We can fix that with https://github.com/WordPress/twentytwentyfour/pull/516

**Review by luminuu — APPROVED** [review]

**Pull request merged**